src/main/scala/scorex/core/network/message/BasicMessagesRepo.scala

```diff
@@ -143,7 +145,7 @@ class ModifiersSpec(maxMessageSize: Int) extends MessageSpecV1[ModifiersData] wi
 
     val (msgCount, msgSize) = modifiers.foldLeft((0, HeaderLength)) { case ((c, s), (_, modifier)) =>
       val size = s + NodeViewModifier.ModifierIdSize + 4 + modifier.length
-      val count = if (size <= maxMessageSize) c + 1 else c
+      val count = if (size <= maxMsgSizeWithReserve) c + 1 else c
```

Comment:
I don't like this much, we mess with modifier count information instead of Failing Fast here ... I mean if the load is too big, shouldn't we fail?

Reply:
Agree, at least we need to log. Failing on serialization should be investigated for possible side effects.